app/components/speakers/SpeakersController.js

```diff
@@ -77,6 +87,48 @@ speakersModule.controller('SpeakerDialogController',
                 {field: 'Position', value: sdc.speaker.position});
         }
 
+        sdc.count = function(speaker, sessions){
+            var count = 0;
+            for(var i = 0; i < sessions.length; i++) {
+                if(sessions[i].speakers.length > 0) {
+                    for(var j = 0; j < sessions[i].speakers.length; j++) {
+                        if(speaker.id == sessions[i].speakers[j].id)
+                            count++;
```

Comment:
This function can be replaced by [Array:filter][1] 

```js
sessions.filter(function(session){
     return session.speakers.filter(function(sp){
              return sp.id == 291;
     })
    .length > 0;
})
```
e.g. to search for all sessions by speaker id 291. More concise and clear.

[1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter